libGeoIP/GeoIP.c

```diff
@@ -861,7 +861,7 @@ char * _GeoIP_iso_8859_1__utf8(const char * iso)
     char * p;
```

Comment:
I wonder if we should drop this commit and just make a separate PR where we switch to `clang-format` with the style used in `geoipupdate`.

Reply:
Yeah, it was a bit huge. I've dropped it. There were a couple conflicts after doing so (the comments typo commit and the last one) but they were minor.